src/main/java/formflow/library/controllers/FormFlowErrorController.java

```diff
@@ -0,0 +1,88 @@
+package formflow.library.controllers;
+
+import static org.springframework.boot.web.error.ErrorAttributeOptions.Include.BINDING_ERRORS;
+import static org.springframework.boot.web.error.ErrorAttributeOptions.Include.EXCEPTION;
+import static org.springframework.boot.web.error.ErrorAttributeOptions.Include.MESSAGE;
+import static org.springframework.boot.web.error.ErrorAttributeOptions.Include.STACK_TRACE;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Arrays;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.error.ErrorAttributeOptions;
+import org.springframework.boot.web.servlet.error.ErrorAttributes;
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class FormFlowErrorController implements ErrorController {
+
+  private final boolean showStackTrace;
+  private final String errorPath;
+
+  private final String prettyPrintPackages;
+  private final ErrorAttributes errorAttributes;
+
+  public FormFlowErrorController(
+      @Value("${form-flow.error.show-stack-trace:false}") boolean showStackTrace,
+      @Value("${server.error.path:${error.path:/error}}") String errorPath,
```

Comment:
Is this default of `${error.path:/error}` - where is `error.path` to be defined? Is it a system thing?

Reply:
This one's bad copy paste, thanks for catching that!

Reply:
@bencalegari what was this suppose to be? Right now it's only hitting the default (which is working fine). Do you mean `server.error.path` should be something like `form-flow.error.path`?

Reply:
I wanted `@Value("${server.error.path:/error")`, so it can be overridden to a different error path if you wanted to implement your own.

Reply:
Updated!